azext_edge/edge/providers/check/mq.py

```diff
@@ -495,26 +497,53 @@ def evaluate_brokers(
                 ),
             )
 
-            for pod in [
-                AIO_BROKER_DIAGNOSTICS_PROBE_PREFIX,
-                AIO_BROKER_FRONTEND_PREFIX,
-                AIO_BROKER_BACKEND_PREFIX,
-                AIO_BROKER_AUTH_PREFIX,
-                AIO_BROKER_HEALTH_MANAGER,
-                AIO_BROKER_DIAGNOSTICS_SERVICE,
-                AIO_BROKER_OPERATOR,
-                AIO_BROKER_FLUENT_BIT,
+            pods: List[dict] = []
+
+            for prefix, label in [
+                (AIO_BROKER_DIAGNOSTICS_PROBE_PREFIX, MQ_NAME_LABEL),
+                (AIO_BROKER_FRONTEND_PREFIX, MQ_NAME_LABEL),
+                (AIO_BROKER_BACKEND_PREFIX, MQ_NAME_LABEL),
+                (AIO_BROKER_AUTH_PREFIX, MQ_NAME_LABEL),
+                (AIO_BROKER_HEALTH_MANAGER, MQ_NAME_LABEL),
+                (AIO_BROKER_DIAGNOSTICS_SERVICE, MQ_NAME_LABEL),
+                (AIO_BROKER_OPERATOR, MQ_NAME_LABEL),
+                (AIO_BROKER_FLUENT_BIT, MQ_NAME_LABEL),
```

Comment:
Just a note - and this might change in the future, but I think for now we can probably remove the label from these tuples since they're all using the same one

Reply:
oh got catch, will update!